modules/nubaBidAdapter.js

```diff
@@ -0,0 +1,19 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { BANNER, NATIVE, VIDEO } from '../src/mediaTypes.js';
+import { isBidRequestValid, buildRequests, interpretResponse } from '../libraries/teqblazeUtils/bidderUtils.js';
+
+const BIDDER_CODE = 'nuba';
+
+const AD_URL = 'https://ads.nuba.io/openrtb2/auction';
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [BANNER, VIDEO, NATIVE],
+
+  isBidRequestValid: isBidRequestValid(),
+  buildRequests: buildRequests(AD_URL),
+  interpretResponse,
+  getUserSyncs: [],
```

Comment:
This is a type error. The getUserSyncs property must be a function returning an array or undefined. If you don’t need it, remove the property; otherwise, define it as a function that returns an empty array. 

ref: type for [spec.getUserSync](https://github.com/prebid/Prebid.js/blob/ee900eb648da036cc53c62a00ac23d35eba25aff/src/adapters/bidderFactory.ts#L150)

console error: 

<img width="521" height="168" alt="Image" src="https://github.com/user-attachments/assets/cf84deec-99aa-4ebe-a61c-8db034a37f51" />